alibuild_helpers/build.py

```diff
@@ -709,8 +710,13 @@ def doBuild(args, parser):
                         a=args.architecture,
                         p=spec["package"])
 
+          getstatusoutput('cd {}; {} -xzf {}'.format(workDir, tar(), d))
           getstatusoutput("ln -snf %s %s" % (src, dst1))
           getstatusoutput("ln -snf %s %s" % (src, dst2))
+
+          os.environ['WORK_DIR'] = workDir
```

Comment:
This will set WORK_DIR for the whole script, which we do not want. Can you do it inside the getstatusoutput?